pyvisgen/simulation/scan.py

```diff
@@ -75,13 +75,13 @@ def calc_fourier(img, bas, lm, spw_low, spw_high):
         -2
         * pi
         * 1j
-        * (ul / c.value * spw_low + vm / c.value * spw_low + wn / c.value * spw_low)
+        * (ul + vm + wn) / 3e8 * spw_low
```

Comment:
We could change the value of c to the value given by `scipy.constants` though

Reply:
done